engines/bastion_katello/app/assets/javascripts/bastion_katello/products/views/products.html

```diff
@@ -41,6 +41,10 @@ <h2 translate>Products</h2>
         </li>
 
         <li role="menuitem">
+          <a ng-click="openAdvancedSyncModal()" ng-show="permitted('edit_products')" translate>Advanced Sync</a>
```

Comment:
Yeah, i considered during the single repo but i figured this was the 2% case, and the main sync was the 98% case.  I'd probably prefer to keep it consistent with the single repo case.